svm/tests/example-programs/transfer-from-account/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+use solana_program::{
+    account_info::{AccountInfo, next_account_info}, entrypoint, entrypoint::ProgramResult, pubkey::Pubkey,
+    program::invoke, system_instruction,
+};
+
+entrypoint!(process_instruction);
+
+
+fn process_instruction(
+    _program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    _data: &[u8]
+) -> ProgramResult {
+    let accounts_iter = &mut accounts.iter();
+    let payer = next_account_info(accounts_iter)?;
+    let recipient = next_account_info(accounts_iter)?;
+    let data_account = next_account_info(accounts_iter)?;
+    let system_program = next_account_info(accounts_iter)?;
+
+    let amount = u64::from_le_bytes(data_account.data.borrow()[0..8].try_into().unwrap());
+
+    invoke(
+        &system_instruction::transfer(payer.key, recipient.key, amount),
+        &[payer.clone(), recipient.clone(), system_program.clone()],
+    )?;
```

Comment:
I'm wondering if we should avoid doing any CPI in here, just for benchmarking purposes. If we keep the call stack to one, we can benchmark the SVM using the VM metrics (create, serialize, etc.) to get an understanding of what the API can handle. What do you think?

Reply:
Anything program that utilizes read-only accounts and writable accounts can work. This is the simplest example I had in my mind whose results are easy to verify, but I can also use another program.

The test I created in particular isn't useful for benchmarking since it leverages shuttle scheduling mechanism, instead of that of Rust.

Reply:
If we're not interesting in benchmarking it then yep, this is fine as-is!